docs/source/usersguide/depletion.rst

```diff
@@ -370,3 +370,57 @@ The values of the one-group microscopic cross sections passed to
 :class:`openmc.deplete.IndependentOperator` are fixed for the entire depletion
 simulation. This implicit assumption may produce inaccurate results for certain
 scenarios.
+
+MSR-Continuous depletion
+===============================
+The :class:`~openmc.deplete.MsrContinuous` class adds the capability to add
+nuclide removal rates to a depletable material.
+
+An instance of this class requires
+:class:`~openmc.deplete.abc.TransportOperator` and :class:`~openmc.Model`
+instances, and can be passed directly to one of the Integrator classes::
+
+    ...
+    msr = openmc.deplete.msr.MsrContinuous(op, model)
+    integrator = openmc.deplete.PredictorIntegrator(op, time_steps, power,
+                msr_continuous=msr)
+
+Defining removal rates
+----------------------
+
+Removal rates can be added to a :class:`~openmc.deplete.MsrContinuous` instance
+with the :meth:`~openmc.deplete.MsrContinuous.set_removal_rate()` method.
+Depletable materials from which nuclides are removed can be specified using the
+:class:`~openmc.Material` instance previously defined. Material id or
+name are also accepted entries.
+Removal rate default units are `1/s`, but `1/h` or `1/d` can also be used.
+For example, to set a Xenon removal from `mat1` with a cycle time of `10 sec`
+(removal rate of 0.1 `1/s`), you'd use::
+
+    mat1 = openmc.Material(material_id=1, name='fuel')
+
+    ...
+
+    # by openmc.Material object
+    msr.set_removal_rate(mat1, ['Xe'] 0.1)
+    # or by material id
+    msr.set_removal_rate(1, ['Xe'] 0.1)
+    # or by material name
+    msr.set_removal_rate('fuel', ['Xe'] 0.1)
+
+Note that in this case the nuclides that are removed will not be tracked.
+
+Defining destination material
+-----------------------------
+
+To keep track of the nuclides that are removed or to define a feed from one
+depletable material to another, a `dest_mat` argument needs to be passed to the
+:meth:`~openmc.deplete.MsrContinuous.set_removal_rate()` method. For example,
+to remove nuclides from `mat1` and feed to `mat2`, you'd use::
+
+    ...
+    mat2 = openmc.Material(name='storage')
+
+    ...
+
+    msr.set_removal_rate(mat1, ['Xe'] 0.1, dest_mat=mat2)
```

Comment:
```suggestion
To keep track of the nuclides that are removed or to define a feed from one
depletable material to another, a `destination_material` argument needs to be passed to the
:meth:`~openmc.deplete.MsrContinuous.set_removal_rate()` method. For example,
to remove nuclides from `mat1` and feed to `mat2`, you'd use::

    ...
    mat2 = openmc.Material(name='storage')

    ...

    msr.set_removal_rate(mat1, ['Xe'] 0.1, destination_material=mat2)
```
Consider using `destination_material` instead of `dest_mat`. I think this is more consistent with the current style of naming variables in OpenMC.